django/core/management/commands/runserver.py

```diff
@@ -69,7 +82,9 @@ def handle(self, *args, **options):
         if not settings.DEBUG and not settings.ALLOWED_HOSTS:
             raise CommandError('You must set settings.ALLOWED_HOSTS if DEBUG is False.')
 
-        self.use_ipv6 = options['use_ipv6']
+        self.verbosity = options.get("verbosity", 1)
```

Comment:
Please cleanup `options` usage as done in 1845bc1d1007751a7f65c66aeddc35f032f6bf41 throughout the new management commands and in the edits.